tests/Feature/SampleFeatureTest.php

```diff
@@ -2,23 +2,30 @@
 
 namespace Saritasa\LaravelTestbed\Tests\Feature;
 
+use Illuminate\Database\Eloquent\Collection;
 use Mockery;
+use PHPUnit\Framework\AssertionFailedError;
+use Saritasa\LaravelTestbed\Traits\ApiListSortingCheck;
 use TestApp\Services\MyService;
 
 class SampleFeatureTest extends TestCase
```

Comment:
That's definitely not a "SampleFeatureTest" but a true "ApiListSortingCheckTest". A very common practice to name test as "TestedClass" + "Test".

Reply:
done.